### DISCUSSION TITLE
How to publish a .msix app using the CLI?

### DISCUSSION THREAD

**Discussion by jakubmeysner** [microsoft/msstore-cli#17] [Q&A]
How do you actually publish an .msix app to the MS Store using the CLI? Commands related to updating a submission or publishing a draft submission seem to be documented pretty well, commands related to actually creating a submission or adding packages to it - not so much. Running `msstore init` or `msstore publish` just returns a `We could not find a project configurator for the project` error. How do you set up a project configurator for the project? `msstore reconfigure` does not seem to allow you to do that.

**Comment by azchohfi** [comment]
It depends on the type of project you are trying to publish.
We don't support publishing a loose MSIX just yet. What type of project are you trying to publish?
https://learn.microsoft.com/en-us/windows/apps/publish/msstore-dev-cli/init-command
Still, support for MSIX is coming.

**Reply by jakubmeysner** [reply]
I already have a process in place for building the MSIX, I'd like just to be able to publish it to the MS Store in CD.

**Comment by NickRimmer** [comment]
Is it still not supported to publish msix with cli?

**Reply by azchohfi** [reply]
It does support MSIX for a few months now:
`msstore publish PATH_TO_MSIX_DIRECTORY`

**Reply by NickRimmer** [reply]
Thank you, will investigate what this `PATH_TO_MSIX_DIRECTORY` should be \~_\~
